Corona-Warn-App/src/main/java/de/rki/coronawarnapp/CoronaWarnApplication.kt

```diff
@@ -138,6 +140,7 @@ class CoronaWarnApplication : Application(), HasAndroidInjector {
         Timber.v("Setting up test result available notification services.")
         pcrTestResultAvailableNotificationService.setup()
         raTestResultAvailableNotificationService.setup()
+        testCertificateRetrievalScheduler.setup()
```

Comment:
This makes the `TestCertificateRepository` active, with this nothing would happen.